docs/config/README.md

```diff
@@ -877,11 +877,20 @@ The module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Variable   | Default        | Description                                            |
-| ---------- | -------------- | ------------------------------------------------------ |
-| `symbol`   | `"☕ "`         | The symbol used before displaying the version of Java. |
-| `style`    | `"dimmed red"` | The style for the module.                              |
-| `disabled` | `false`        | Disables the `java` module.                            |
+| Variable   | Default                             | Description                                     |
```

Comment:
```suggestion
| Option     | Default                             | Description                                     |
```
To avoid naming conflicts with variable in format string